packages/workbox-webpack-plugin/package.json

```diff
@@ -23,7 +23,8 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
-    "workbox-build": "^2.0.1"
+    "workbox-build": "^2.0.1",
+    "workbox-sw": "^2.0.1"
```

Comment:
It's worrying if you added dep for workbox-sw and it can't be copied over :-S

Reply:
I wonder whether the issue you described was due to `lerna`-being-`lerna`.

If you do a `gulp lerna-bootstrap` in the top-level `workbox/` directory that might take care of ensuring that `workbox-build`'s dependency on `workbox-sw` gets satisfied, without you having to explicitly list it.